src/Storages/Kafka/StorageKafka2.cpp

```diff
@@ -132,6 +132,7 @@ StorageKafka2::StorageKafka2(
     , WithContext(context_->getGlobalContext())
     , keeper(getContext()->getZooKeeper())
     , keeper_path((*kafka_settings_)[KafkaSetting::kafka_keeper_path].value)
+    , fs_keeper_path(keeper_path)
```

Comment:
I made this change for a different implementation, but I liked this, so kept it even when I implemented the logic without this.